R/visit_structure.R

```diff
@@ -76,12 +76,16 @@ visit_structure <- function(x, sorted = TRUE) {
 #' @rdname visit_structure
 #' @param ... further parameters
 #' @export
+#' @return plot of the visit plan
 plot.secuTrialvisit <- function(x, ...) {
   # construct the figure.
   z <- !is.na(as.matrix(x[, -1]))
   names <- gsub("tmpvar.", "", names(x[, -1]))
-  paropts <- par()
-  on.exit(paropts)
+
+  # adjusted as per request in CRAN review
```

Comment:
without the comment?

Reply:
If you feel strongly about this I can remove it but generally I like comments. There must have been a reason why it was done differently at first, so maybe someone may wonder "Why are you doing it like this?" The comment would clarify that.

Reply:
I don't feel strongly about it, but it's such a minor thing that I dont think the comment helps. The git comment/PR make it clear that it's due to the CRAN review... i dont think that comment in the code helps as it doesnt actually say anything about what the code itself does...